flyway-core/src/main/java/org/flywaydb/core/Flyway.java

```diff
@@ -906,6 +930,23 @@ public void setSkipDefaultResolvers(boolean skipDefaultResolvers) {
     }
 
     /**
+     * <p>Performs a migration of the database and then rolls back the migration.</p>
+     *
+     * <p>This is an alias for:</p>
+     * <pre class="brush:java"><code>
+     *  flyway.setDryRun(true);
+     *  flyway.migrate();
+     * </code></pre>
+     *
+     * @return The number of migrations that would have been successfully applied
+     * @throws FlywayException when the migration would have failed or Dry-run is not supported by the database driver
+     */
+    public int dryRun() throws FlywayException {
+        setDryRun(true);
+        return migrate();
+    }
```

Comment:
Any command is 'dryrun'able; however, one sees 'migrate' as the obvious candidate for what a 'dry run' is.  I suppose there could be a dryrun of a repair, for example which would be accomplished by:

```java
flyway.setDryRun(true);
flyway.repair();
```